### PR TITLE
Sync with cnode

### PR DESCRIPTION
**Pull request by pjenys0**
Sync with new change of cNode at https://github.com/pastelnetwork/pastel/pull/62

### CODE REVIEW

File: pastel/tickets.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ type TradeTicketData struct {
 	PastelID         string `json:"pastelID"`          // PastelID of the buyer
 	SellTXID         string `json:"sell_txid"`         // txid with sale ticket
 	BuyTXID          string `json:"buy_txid"`          // txid with buy ticket
-	ArtTXID          string `json:"art_txid"`          // txid with either 1) art activation ticket or 2) trade ticket in it
+	NFTTXID          string `json:"NFT_txid"`          // txid with either 1) NFT activation ticket or 2) trade ticket in it
```
Comment (leonidcode20) [inline pastel/tickets.go:17]: tag imo should be `nft_txid`

File: proto/walletnode/register_artwork.proto [hunk 2]
```diff
@@ -91,23 +91,23 @@ message EncoderParameters {
   bytes oti = 1;   //12 bytes
 }
 
-message SendSignedArtTicketRequest {
-    bytes art_ticket = 1;
-    bytes artist_signature = 2;
+message SendSignedNFTTicketRequest {
+    bytes NFT_ticket = 1;
```
Comment (leonidcode20) [inline proto/walletnode/register_artwork.proto:95]: `nft_ticket` make more sense ans is more compliance to protbuf naming guide.

File: pastel/tickets.go (hunk 1 above)
Comment (pjenys0) [reply pastel/tickets.go:17]: this is to follow cnode change.

**Review by georgkcode0 — DISMISSED** [review]

**Review by danileng20 — APPROVED** [review]
LGTM!

**Review by leonidcode20 — APPROVED** [review]
LGTM!

**Pull request merged**